common/djangoapps/util/tests/test_db.py

```diff
@@ -31,65 +27,6 @@ class TransactionManagersTestCase(TransactionTestCase):
 
     To test do: "./manage.py lms --settings=test_with_mysql test util.tests.test_db"
     """
-    DECORATORS = {
-        'outer_atomic': outer_atomic(),
-        'outer_atomic_read_committed': outer_atomic(read_committed=True),
-    }
-
-    @ddt.data(
-        ('outer_atomic', IntegrityError, None, True),
-        ('outer_atomic_read_committed', type(None), False, True),
-    )
-    @ddt.unpack
-    def test_concurrent_requests(self, transaction_decorator_name, exception_class, created_in_1, created_in_2):
-        """
-        Test that when isolation level is set to READ COMMITTED get_or_create()
-        for the same row in concurrent requests does not raise an IntegrityError.
-        """
-        transaction_decorator = self.DECORATORS[transaction_decorator_name]
-        if connection.vendor != 'mysql':
-            raise unittest.SkipTest('Only works on MySQL.')
```

Comment:
@symbolist haven't tried yet. I asked one of my teammates to try it. Aligning with my comment above, should this be running in the complete testsuite?

Reply:
@jfavellar90 There is no need for this test to exist anymore since the relevant code is being removed.

I ran this test locally using the following steps (I recommend replicating it to confirm the expected behavior):

1. Give db user permission to create test db
```
make dev.dbshell.edxapp

mysql> GRANT ALL PRIVILEGES ON *.* TO 'edxapp001'@'%';
Query OK, 0 rows affected (0.00 sec)

mysql> SHOW GRANTS FOR 'edxapp001';
+------------------------------------------------------------+
| Grants for edxapp001@%                                     |
+------------------------------------------------------------+
| GRANT ALL PRIVILEGES ON *.* TO 'edxapp001'@'%'             |
| GRANT ALL PRIVILEGES ON `edxapp`.* TO 'edxapp001'@'%'      |
| GRANT ALL PRIVILEGES ON `edxapp_csmh`.* TO 'edxapp001'@'%' |
+------------------------------------------------------------+
3 rows in set (0.00 sec)

```

2. Set tests to use the devstack db

```
diff --git a/setup.cfg b/setup.cfg
index 954ce7b818..5c5c419dd2 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -1,5 +1,5 @@
 [tool:pytest]
-DJANGO_SETTINGS_MODULE = lms.envs.test
+DJANGO_SETTINGS_MODULE = lms.envs.devstack_docker
```

3. Run tests

```
> pytest common/djangoapps/util/tests/test_db.py::TransactionManagersTestCase

============================================================================== short test summary info =====
FAILED common/djangoapps/util/tests/test_db.py::TransactionManagersTestCase::test_concurrent_requests_1 - AssertionError: assert False
===================================================================== 1 failed, 3 passed, 21 warnings in 34.49s =====
```

Reply:
@symbolist after following the steps, I can confirm the same test results. As you had mentioned in your first comment, the first test fails, the rest pass, so we are OK on this side.